grouper/fe/templates.py

```diff
@@ -86,14 +87,42 @@ def _get_form_alerts(self, errors: Dict[str, List[str]]) -> List[Alert]:
 @dataclass(repr=False, eq=False)
 class PermissionTemplate(BaseTemplate):
     permission: Permission
-    group_grants: List[GroupPermissionGrant]
-    service_account_grants: List[ServiceAccountPermissionGrant]
     access: PermissionAccess
     audit_log_entries: List[AuditLogEntry]
 
     template: InitVar[str] = "permission.html"
 
 
+@dataclass(repr=False, eq=False)
+class PermissionGroupGrantsTemplate(BaseTemplate):
+    permission: Permission
+    access: PermissionAccess
+    grants: List[GroupPermissionGrant]
+    audit_log_entries: List[AuditLogEntry]
```

Comment:
Slight potential for confusion because the audit log entries will be for the permission generally (so not just for grants to groups), but that's the least of the problems with the audit log :D